src/button/_button.scss

```diff
@@ -277,7 +287,6 @@ input.mdl-button[type="submit"] {
     &.mdl-button--disabled.mdl-button--disabled {
       background-color: $button-primary-color-disabled;
       color: $button-secondary-color-disabled;
-      @include shadow-2dp();
```

Comment:
Up, thanks for catching that. Accidentally pushed a diff commit up by not branching before starting new work. :( Easy enough fix!